lemarche/templates/tenders/detail_card.html

```diff
@@ -72,9 +72,28 @@ <h2>Montant du marché</h2>
                 {% if tender.author == request.user or user_has_contact_click_date or user_can_display_tender_contact_details %}
                     {% include "tenders/_detail_contact.html" with tender=tender %}
                 {% elif user.kind == user.KIND_PARTNER %}
-                    <p>
-                        Contactez Sofiane ou Abdessamad via le chat de discussion qui se trouve en bas à droite du site pour être mis en relation avec l'acheteur.
-                    </p>
+                    <div class="alert alert-info" role="alert">
+                        <p>
```

Comment:
```suggestion
                        <p class="mb-1">
```
pour homogénéiser avec les autres alerts